assets/js/app.js

```diff
@@ -310,7 +349,20 @@ function App({ uniqueId, socket }) {
 
     return (
         <IntlProvider locale={language} messages={messages}>
-            {machineState && boxConfig && (
+            {currentTime && <div className="current-time">{currentTime.getHours()}:{currentTime.getMinutes()}</div>}
```

Comment:
Should we add seconds to the display? If the primary function of the clock is to signal to users that the application is not frozen, this might be better.

Reply:
Agree...

Reply:
Fixed